src/ray/stats/metric_defs.cc

```diff
@@ -145,6 +145,12 @@ DEFINE_stats(placement_groups,
 /// ===================== INTERNAL SYSTEM METRICS =================================
 /// ===============================================================================
 
+DEFINE_stats(io_context_event_loop_lag_ms,
+             "Latency of a task from post to execution",
```

Comment:
This is basically the queuing time we print in the debug state right?

Reply:
I think so. just that one does not have a metrics and we don't always want to look at logs.